settings.py

```diff
@@ -115,45 +114,50 @@ def get_from_env(var, default):
 SECURE_BROWSER_XSS_FILTER = True
 SECURE_CONTENT_TYPE_NOSNIFF = True
 
-# List of callables that know how to import templates from various sources.
-TEMPLATE_LOADERS = (
-    'django.template.loaders.filesystem.Loader',
-    'django.template.loaders.app_directories.Loader'
-)
+SILENCED_SYSTEM_CHECKS = ['urls.W002']
 
-MIDDLEWARE_CLASSES = (
+MIDDLEWARE = [
     # make all things SSL
     #'sslify.middleware.SSLifyMiddleware',
 
     # secure a bunch of things
-    'djangosecure.middleware.SecurityMiddleware',
+    'django.middleware.security.SecurityMiddleware',
```

Comment:
With this change, is `'djangosecure',` in `INSTALLED_APPS` still needed? Helios seems to work without it, so it's probably unused - but still, removing something with "secure" in its name feels a bit scary ;-)

More important: Thanks for this pull request - it looks like it might help to save my ass ^W^W^W get helios running on openSUSE Leap 15.1 :-)

Reply:
That's what actually worried me, I still didn't have time to check it out thoroughly

Reply:
Indeed, that isn't necessary anymore, along with django-sslify!